### PR TITLE
Feature: Spectrum addition

### PR DESCRIPTION
**Pull request by k9ert**
This PR is adding the Spectrum Extension to the Specter Release. Some bugs along the way are solved, too:
* dependency for build as the linux build seem to be quicker nowadays and we can't no longer use parallel building (but was quicker in the past)
* the language-version in pre-commit is an accidental change and need to be removed. Therefore, this is a draft-PR so far
* A bug in main.js prevented the addition of cli params
* Extension now can bring additional devices but they are not packed in the pyinstaller binary. THerefore, they need to explicitely discovered in parallel with the controller and the config of each extensions.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -3,4 +3,3 @@ repos:
     rev: 22.3.0
     hooks:
     - id: black
-      language_version: python3.9
```
Comment (k9ert) [inline .pre-commit-config.yaml:6]: Don't change this

File: requirements.in [hunk 2]
```diff
@@ -33,3 +33,4 @@ pytimeparse==1.1.8
 cryptoadvance-liquidissuer==0.2.4
 specterext-exfund==0.1.7
 specterext-faucet==0.1.2
+cryptoadvance.spectrum==0.2.0rc5
```
Comment (k9ert) [inline requirements.in:36]: THis shouldn't be a rc to merge into master

**Review by moneymanolis — APPROVED** [review]
All good, just don't change the pre-commit yml :-)

**Comment by moneymanolis** [comment]
@k9ert Having `activate_spectrum_node()` in the callback of the Spectrum service leads to the fact that the active node from the config is not used but ALWAYS the Spectrum Node. I would just delete the below line. I don't see why it is necessary:
[link to code part](https://github.com/cryptoadvance/spectrum/blob/4ffb4af6ab242f79d19006391c2e615c93e8575d/src/cryptoadvance/specterext/spectrum/service.py#L64)

**Comment by moneymanolis** [comment]
TODOS / bugs / quirks:
- [ ] If the Spectrum node is set up but then changed in a way that the connection can't be established (wrong port etc.) you still get the success screen.

**Comment by moneymanolis** [comment]
### Visuals

Together with[ this PR in the Spectrum repo](https://github.com/cryptoadvance/spectrum/pull/12), the setup of Spectrum would look like this now:

Case 1 - success:
https://user-images.githubusercontent.com/70536101/204643747-5644ad23-c18d-4e2c-ad2e-9511d9d46fd9.mov

Case 2 - wrong config, then changed to successful connection:
https://user-images.githubusercontent.com/70536101/204643873-dffe5f54-6042-450e-bcf4-7478e538cc89.mov

Case 3 - running Bitcoin Core node on the same network as the Spectrum node is not supported in the alpha version (creates problems with the wallet manager):
https://user-images.githubusercontent.com/70536101/204644168-05a75840-99ac-487e-8fb2-1dd38bdea51c.mov

**Review by moneymanolis — COMMENTED** [review]
All good!

**Pull request merged**